### PR TITLE
161348754 Users should be able to presented with a reset pa…

### PR DESCRIPTION
**Pull request by dnuwa**
#### What does this PR do?
Adds feature for resetting a password

#### Description of Task to be completed?
- Get the user email that requires a password reset
- Add email input form
- Add a password form with 2 input field (enter a new password and confirm password)
#### How should this be manually tested?
- Go to the password reset URL: `/passwordreset` 
- You should be able to enter your email from which you will receive an email on successful submission
- Click on the link sent to your email to reset the password

#### Any background context you want to provide?
Only users with existing accounts can reset password

#### What are the relevant pivotal tracker stories?
[#161348754](https://www.pivotaltracker.com/story/show/161348754)

#### Screenshots (if appropriate)
Password reset page
image

Redirect from the email sent
image

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+engines:
+  duplication:
+    enabled: true
+    config:
+      languages:
+        javascript:
+          mass_threshold: 75
```
Comment (RachaelNantale) [inline .codeclimate.yml]: Please add a new line at the end of file

File: src/components/Login/index.js [hunk 2]
```diff
@@ -45,26 +47,12 @@ export const Login = ({ onChange, onSubmit }) => {
...
-                <SocialLogin />
-              </div>
-            </div>
+          {loginSigninButtons('signin-buttons', 'btn btn-primary sign-in-bt', 'signInBt', 'signInBt', 'Sign In')}
+          <div className="d-flex justify-content-around align-items-end">
+            {generateRedirectLinks('', 'passwordreset', 'Forgot password?')}
+            {generateRedirectLinks('', 'signup', 'Not yet a member?')}
           </div>
-          {generateRedirectLinks('Not yet a member? ', 'signup', 'Sign Up')}
+          {/* {generateRedirectLinks('Not yet a member? ', 'signup', 'Sign Up')} */}
```
Comment (RachaelNantale) [inline src/components/Login/index.js]: why did you comment this out?

**Review by brucemakallan — DISMISSED** [review]
Looks good

**Review by judeinno — DISMISSED** [review]

File: src/components/Login/index.js (hunk 2 above)
Comment (dnuwa) [reply src/components/Login/index.js]: @RachaelNantale thanx for pointing this out.
This was duplicated so it needs to be removed which I've just done

**Review by brucemakallan — DISMISSED** [review]
Thanks for making the requested changes

File: src/components/Login/index.js (hunk 2 above)
Comment (RachaelNantale) [reply src/components/Login/index.js]: Your welcome

**Review by RachaelNantale — DISMISSED** [review]
Good job working on the changes i requested.

**Review by judeinno — DISMISSED** [review]

**Review by Sulaiman-Mozes — DISMISSED** [review]

**Review by brucemakallan — DISMISSED** [review]

**Review by RachaelNantale — DISMISSED** [review]

**Review by judeinno — DISMISSED** [review]

**Review by brucemakallan — DISMISSED** [review]

**Review by Sulaiman-Mozes — DISMISSED** [review]

**Review by judeinno — APPROVED** [review]

**Review by RachaelNantale — APPROVED** [review]

**Review by brucemakallan — APPROVED** [review]

**Pull request merged**